docs/neo3_guides/token_transfer.md

```diff
@@ -146,4 +145,28 @@ in [this Section](neo3_guides/contract_invocation.md#signing-a-transaction-with-
 
 ## Non-fungible Tokens (NEP-11)
 
-?> _In the works_
+neow3j provides a wrapper class `NFToken` that can interact with token contracts that support the [NEP-11](https://github.com/neo-project/proposals/pull/41) standard.
+
+> The NEP-11 standard is not yet final, so there might still be changes to it and there does not exist an implementation for it yet.
```

Comment:
```suggestion
> The NEP-11 standard is not yet final, so there might still be changes to it.

Reply:
I thought we saw some examples somewhere, no?

Reply:
True, I'll remove it.